README.md

```diff
@@ -1,2 +1,40 @@
 # java-globaltracer
-Global ThreadLocal Span for implicit propagation, delegating to another Tracer implementation.
+Global Tracer, forwarding to another Tracer implementation.
+
+This library provides two utility classes `ActiveSpanManager` and `GlobalTracer`:
+
+## ActiveSpanManager
+This class:
+ 1. Provides clients with the `ActiveSpanManger.activeSpan()` method to return the
+    _active span_.
+    If there is no active span, the method never returns a `null` refrence, but
+    a `NoopSpan` instead.
```

Comment:
what does the context of NoopSpan contain? Ex there are some who maintain the propagated state even in noop mode.